### DISCUSSION TITLE
bug: "Go to my inbox" button has invalid filter

### DISCUSSION THREAD

**Discussion by Animalmix55** [Web3Auth/Web3Auth#1324] [Help]
Good evening,

When testing our app, we discovered that utilizing the "Go to my inbox" button when logging in via Gmail through passwordless email login inputs the wrong filter into the Gmail searchbar:

`from=web3auth OR from=torus in:anywhere`
![image](https://user-images.githubusercontent.com/4249734/218647072-f655b035-c6a0-4d8c-9e1b-e58f1a68cb25.png)

versus

`from=web3auth OR from=tor.us in:anywhere` 
![image](https://user-images.githubusercontent.com/4249734/218647192-c5e153b6-25e3-489c-bbad-13a1c4c8c76d.png)

-Cory

**Comment by YZhenY** [comment]
thanks for reporting this, fixing

**Comment by chaitanyapotti** [comment]
This is fixed now. thanks for reporting the case where it doesn't work as intended for whitelabelled projects

**Reply by Animalmix55** [reply]
Thanks!
